src/test/java/org/broadinstitute/hellbender/tools/walkers/bqsr/BaseRecalibratorIntegrationTest.java

```diff
@@ -162,4 +162,21 @@ public void testBQSRFailWithIncompatibleReference() throws IOException {
                 UserException.IncompatibleSequenceDictionaries.class);
         spec.executeTest("testBQSRFailWithIncompatibleReference", this);
     }
+
+    @Test
+    public void testBQSRFailWithIncompatibleSequenceDictionaries() throws IOException {
+        final String resourceDir =  getTestDataDir() + "/" + "BQSR" + "/";
+
+        final String bam_chr20 = resourceDir + WGS_B37_CH20_1M_1M1K_BAM;
+        final String dbSNPb37_chr17 =  getResourceDir() + "dbsnp_132.b37.excluding_sites_after_129.chr17_69k_70k.vcf";
+
+        final String  NO_ARGS = "";
+        final BQSRTest params = new BQSRTest(b37_reference_20_21, bam_chr20, dbSNPb37_chr17, NO_ARGS, resourceDir + "expected.txt");
+        IntegrationTestSpec spec = new IntegrationTestSpec(
+                params.getCommandLine(),
+                1,
+                UserException.IncompatibleSequenceDictionaries.class);
+        spec.executeTest("testBQSRFailWithIncompatibleReference", this);
+    }
```

Comment:
Test case is still mis-named as `testBQSRFailWithIncompatibleReference` despite the comment above -- should be `testBQSRFailWithIncompatibleSequenceDictionaries`